cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -105,6 +105,129 @@
       </select>
       <span class="tip tip-stacked">Identify the course language here. This is used to assist users find courses that are taught in a specific language.</span>
     </li>
+
   </ol>
 </section>
+
+<section class="group-settings marketing">
+  <header>
+    <h2 class="title-2">${_("Introducing Your Course")}</h2>
+    <span class="tip">${_("Information for prospective students")}</span>
+  </header>
+  <ol class="list-input">
+
+    <li class="field text" id="field-course-title">
+      <label for="course-title">${_("Course Title")}</label>
+      <input type="text" id="course-title" data-display-name="${context_course.display_name}">
+      <span class="tip tip-stacked">${_("Displayed as title on the course details page. Limit to 50 characters.")}</span>
+    </li>
+    <li class="field text" id="field-course-subtitle">
+      <label for="course-subtitle">${_("Course Subtitle")}</label>
+      <input type="text" id="course-subtitle">
+      <span class="tip tip-stacked">${_("Displayed as subtitle on the course details page. Limit to 150 characters.")}</span>
+    </li>
+    <li class="field text" id="field-course-duration">
+      <label for="course-duration">${_("Course Duration")}</label>
+      <input type="text" id="course-duration">
+      <span class="tip tip-stacked">${_("Displayed on the course details page. Limit to 50 characters.")}</span>
+    </li>
+    <li class="field text" id="field-course-description">
+      <label for="course-description">${_("Course Description")}</label>
+      <textarea class="text" id="course-description"></textarea>
+      <span class="tip tip-stacked">${_("Displayed on the course details page. Limit to 1000 characters.")}</span>
+    </li>
+
+    <li class="field image" id="field-course-image">
+      <label for="course-image-url">${_("Course Card Image")}</label>
+      <div class="current current-course-image">
+        % if context_course.course_image:
+        <span class="wrapper-course-image">
+          <img class="course-image" id="course-image" src="${course_image_url}" alt="${_('Course Image')}"/>
+        </span>
+
+        <span class="msg msg-help">
+        ${_("You can manage this image along with all of your other <a href='{}'>files &amp; uploads</a>").format(upload_asset_url)}
+        </span>
+
+        % else:
+        <span class="wrapper-course-image">
+          <img class="course-image placeholder" id="course-image" src="${course_image_url}" alt="${_('Course Image')}"/>
+        </span>
+        <span class="msg msg-empty">${_("Your course currently does not have an image. Please upload one (JPEG or PNG format, and minimum suggested dimensions are 375px wide by 200px tall)")}</span>
```

Comment:
Does this mock really need the conditional code? What is actually being used in the test?